### DISCUSSION TITLE
suggestion about the files folder

### DISCUSSION THREAD

**Discussion by Alib234** [ppy/osu#20438] [Ideas] [answered]
My suggestion is that the `Songs` and `Skins` folders should come back and then if lazer really needs it, make **symlinks** of the files, put them in the `files` folder with their sha256 hash as the name in the correct folder according to the first 2 characters of the sha256 hash of the file, aka like lazer already does

**Comment by ghost** [comment]
I'm pretty sure the whole point of moving them to the "files" folder is so they cant be modified directly  by the user

**Comment by peppy** [comment, marked as answer]
As mentioned this defeats the whole purpose. Please check the wiki page on why things are organised as they are.
